go.sum

```diff
@@ -172,6 +172,8 @@ github.com/hashicorp/go-tfe v0.3.16 h1:GS2yv580p0co4j3FBVaC6Zahd9mxdCGehhJ0qqzFM
 github.com/hashicorp/go-tfe v0.3.16/go.mod h1:SuPHR+OcxvzBZNye7nGPfwZTEyd3rWPfLVbCgyZPezM=
 github.com/hashicorp/go-tfe v0.3.17 h1:4n4Z1Kr5hXKI2ntJLyso9yYFHjSBVYRGgK8BG48H6OQ=
 github.com/hashicorp/go-tfe v0.3.17/go.mod h1:SuPHR+OcxvzBZNye7nGPfwZTEyd3rWPfLVbCgyZPezM=
+github.com/hashicorp/go-tfe v0.3.18-0.20190613152246-ce09432e1250 h1:3tzwJ9TtTVHbBLnYZj+QJQlTkx7hic4QlmO4j4ofWCs=
+github.com/hashicorp/go-tfe v0.3.18-0.20190613152246-ce09432e1250/go.mod h1:SuPHR+OcxvzBZNye7nGPfwZTEyd3rWPfLVbCgyZPezM=
```

Comment:
Make sure you execute `go mod tidy` as well to clean up older versions.

Reply:
Thanks, this is a WIP right now as the SDK is not fully merged, but I will remember to do that afterwards. I usually do, but forgot this time.